cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("connecting to db")
+        connection = None
+        while not connection:
+            try:
+                connection = connections["default"]
+                connection.cursor()
+            except OperationalError as e:
+                self.stdout.write(f"{e}, trying again")
```

Comment:
The error message is printed every time an `OperationalError` occurs. It might be helpful to include a counter or a maximum retry limit to avoid infinite loops in case the database is unavailable for an extended period.